README.md

```diff
@@ -1 +1,215 @@
+  <ul>
+    <li><a href="">n/a</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="/">n/a</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **[use linters]**
+- **[Displays a list of books]**
+- **[Allows users to add new books to the list]**
+- **[Allows users to remove books from the list]**
+- **[Use API]**
+
+
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- you need a laptop.
+- you need github installed on it.
+- You need a basic knowledge of Html and Css and JavaScript, and have Basic fondamental of React
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- git clone https://github.com/zbahati/BookStore-React.git
+- cd my-folder
+
+
+### Install
+
+Install this project with:
+
+
+- cd my-project
+- npm install
+- npm start to run the project to the browser
+
+
+### Usage
+
+To use the  BookStore React and Redux website, simply run <span style="color: blue, font-size: 20px;"> npm start<span> command to the Command line of the currecnt directory then React will open the port and the render it to the Browser automatically the you can see the application runing on the browser
+
+
+### Run tests
+
+To run tests, run the following command:
+
+- npx stylelint "**/*.{css,scss}" help to test styles error
+- npx hint . help to test html error
+- npx eslint . help to test jsx error in the react application
+
+### Deployment
+
+You can deploy this project using: github page's, or Render
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Bahati zirimwabagabo**
+
+- GitHub: [zbahati](https://github.com/zbahati)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/zirimwabagabo-bahati);
+- Tweeter : [Twetter](https://twitter.com/b2_bahati);
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[Nav menu toggle on mobile devices]**
+- [ ] **[Add server]**
+- [ ] **[Add books to the Cart]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+show me your support by following me on github and giving me a star please .
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements">Microverse</a>
+
```

Comment:
- [optional] you can remove one acknowledgement header and write everything under one. this will make your Readme professional.